src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -653,6 +691,7 @@ def forward(
             inputs_embeds=inputs_embeds,
             output_hidden_states=output_hidden_states,
             return_dict=return_dict,
+            **kwargs,
```

Comment:
why is this required? it should not. The cache params are passed right above

Reply:
I believe it's the use_cache argument that needs to be passed in for this to work as expected - we could restrict to just passing that through?

Reply:
Have amended this

Reply:
perfect